homepage/migrations/0007_auto_alter_review_field.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2 on 2021-04-25 12:50
+
+import django.core.validators
```

Comment:
```suggestion
from django.core import validators
```
then you can use it simpler in line # 18